src/payment/ProductLineItem.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from '@edx/frontend-i18n';
+
+
+class ProductLineItem extends React.PureComponent {
+  renderSeatType(seatType) {
+    switch (seatType) {
+      case 'professional':
+      case 'no-id-professional':
+        return (<FormattedMessage
+          id="payment.productlineitem.professional.certificate"
+          defaultMessage="Professional Certificate"
+          description="Course certificate type on product details section"
+        />);
+      case 'verified-certificate':
+        return (<FormattedMessage
+          id="payment.productlineitem.verified.certificate"
+          defaultMessage="Verified Certificate"
+          description="Course certificate type on product details section"
+        />);
+      case 'honor':
+      case 'audit':
+      default:
+        return null;
+    }
+  }
+
+  render() {
+    const {
+      product,
+    } = this.props;
+    return (
+      <div className="pb-3">
+        <h2 className="h6">
+          <FormattedMessage
+            id="payment.productlineitem.purchase.cart.heading"
+            defaultMessage="In your cart"
+            description="Heading of the cart in product details section"
+          />
+        </h2>
+        <p>
+          <FormattedMessage
+            id="payment.productlineitem.purchase.cart.subheading"
+            defaultMessage="Your purchase contains the following:"
+            description="Subheading of the cart in product details section"
+          />
+        </p>
+        <div className="row align-items-center">
+          <div className="col-5">
+            <img className="img-thumbnail mb-3" src={product.imgUrl} alt={product.name} />
+          </div>
+          <div className="col-7">
+            <p className="h6">
+              {product.name}
+            </p>
+            <p>{this.renderSeatType(product.seatType)}</p>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+ProductLineItem.propTypes = {
+  product: PropTypes.shape({
```

Comment:
Let's unwrap this >> so the props might look more like `{ imgUrl, name, seatType }` losing the product key.